src/preflight.cts

```diff
@@ -56,3 +56,6 @@ if (process.send) {
 		return result;
 	};
 }
+
+// for some reason process.off does not work
+process.off = process.removeListener;
```

Comment:
Why does it need to work? I don't see it used anywhere in your code. And why is it tsx's responsibility to patch this?

Reply:
Okay, but hows it relevant to this PR?